front/app/modules/commercial/content_builder/admin/components/CraftComponents/Button/index.tsx

```diff
@@ -48,7 +48,13 @@ const Button = ({ text, url, type, alignment }: ButtonProps) => {
       {((enabled && text) || (text && url)) && (
         <ButtonComponent
           linkTo={url}
-          openLinkInNewTab={!url.includes(window.location.hostname)}
+          openLinkInNewTab={
+            url.includes(window.location.hostname)
+              ? false
+              : url.includes('.')
+              ? true
+              : false
```

Comment:
Non-blocking suggestion: I'd avoid using a nested ternary. It makes code harder to read. It might be better to extract out a variable above and use that. https://eslint.org/docs/latest/rules/no-nested-ternary

Reply:
Ah that's a good point. I'll use a variable instead :) Thanks!